src/coreclr/src/debug/debug-pal/win/diagnosticsipc.cpp

```diff
@@ -144,11 +145,14 @@ IpcStream *IpcStream::DiagnosticsIpc::Accept(ErrorCallback callback)
     {
         if (callback != nullptr)
             callback("Failed to GetOverlappedResults for NamedPipe server", ::GetLastError());
-        return nullptr;
+        // clean up the pipe
+        ::CloseHandle(_hPipe);
```

Comment:
Should `_hPipe` be either cleared?  Or possibly reopened?

Reply:
`_hPipe` is cleared below on line 158.